unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Passport/Passport/PlayerInfo/PassportPlayerInfoComponentView.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine.UI;
+using UnityEngine;
+using TMPro;
+
+public class PassportPlayerInfoComponentView : MonoBehaviour, IPassportPlayerInfoComponentView
```

Comment:
In order to follow our UI Components architecture, all our views should implement the `BaseComponentView` class.

Reply:
Additionally, as we talked earlier in the meeting, maybe the "copy-wallet" part would be useful to make it to be part of our common UI Components library.